Resources/Prototypes/Entities/Clothing/Shoes/boots.yml

```diff
@@ -87,5 +87,95 @@
     sprite: Clothing/Shoes/Boots/winterboots.rsi
   - type: Clothing
     sprite: Clothing/Shoes/Boots/winterboots.rsi
-  - type: TemperatureProtection
-    coefficient: 0.2
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterCargo
+  name: cargo winter boots
+  description: Fluffy boots to help survive even the coldest of winters.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootscargo.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootscargo.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterEngi
+  name: engineering winter boots
+  description: Fluffy boots to help survive even the coldest of winters.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootsengi.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootsengi.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterMed
+  name: medical winter boots
+  description: Fluffy boots to help survive even the coldest of winters.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootsmed.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootsmed.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterSci
+  name: science winter boots
+  description: Fluffy boots to help survive even the coldest of winters.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootssci.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootssci.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterSec
+  name: security winter boots
+  description: Fluffy boots to help survive even the coldest of winters.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootssec.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootssec.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterSyndicate
+  name: syndicate's winter boots
+  description: Fluffy boots to help survive even the coldest of winters.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/winterbootssyndicate.rsi
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/winterbootssyndicate.rsi
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterWeb
+  name: web winter boots
+  description: Fluffy boots to help survive even the coldest of winters.
```

Comment:
this needs a custom description, also make sure to thoroughly test the construction